tracing-appender/src/rolling.rs

```diff
@@ -4,12 +4,55 @@ use std::fmt::Debug;
 use std::io;
 use std::path::Path;
 
+/// A file appender with the ability to rotate log files at a fixed schedule.
+///
+/// This struct implements [`std:io::Write` trait][write] and will block on any write operation.
+/// It can be used in conjunction with [`NonBlocking`][non-blocking] to perform writes without
+/// blocking the main thread.
+///
+/// [write]: https://doc.rust-lang.org/nightly/std/io/trait.Write.html
+/// [non-blocking]: ../non_blocking/struct.NonBlocking.html
+///
+/// # Examples
+///
+/// ```rust
+/// # fn docs() {
+/// use tracing_appender::rolling::RollingFileAppender;
+/// use tracing_appender::rolling::Rotation;
+/// use tracing_appender::non_blocking::NonBlocking;
+///
+/// let file_appender = RollingFileAppender::new(Rotation::HOURLY, "/some/directory", "prefix.log");
+/// let (non_blocking, _guard) = NonBlocking::new(file_appender);
+/// # }
+/// ```
 #[derive(Debug)]
 pub struct RollingFileAppender {
     inner: InnerAppender,
 }
 
 impl RollingFileAppender {
+    /// Returns a new `RollingFileAppender`
+    ///
+    /// The returned `RollingFileAppender` will have a fixed rotation whose behavior is
+    /// defined by [`Rotation`](struct.Rotation.html). The provided `directory` and
+    /// `file_name_prefix`, will determine the location and file name prefix of the log file.
```

Comment:
```suggestion
    /// A `RollingFileAppender` will have a fixed rotation whose frequency is
    /// defined by [`Rotation`](struct.Rotation.html). The `directory` and
    /// `file_name_prefix` arguments determine the location and file name's _prefix_
    /// of the log file. `RollingFileAppender` will automatically append the current date
    /// and hour to the file name.
```

How is the current date and hour determined? Is it local time, UTC, etc.?

Reply:
Right now it's UTC. Later on we'll want to add support for accepting other formats.

Reply:
UTC, I'll add a comment in here